### PR TITLE
Common Home Location Info Form + Bug Fixes

### PR DESCRIPTION
**Pull request by cameron-eyds**
*Issue #:* /bcgov/entity#14130

*Description of changes:*
* Created common form for use through the various home location type options
* Implemented field level validations
* Form Events to update to parent, accept validation prompt and emit validation state
* Screens below of example implementations

*NOTE:* 
* Implemented placeholders in Home Location Types where common form will slot in. Still requires event hookups in individual component tickets.

*Added Bug Fixes:*
* Ticket /bcgov/entity#13863 - Fixed global editing bug on leaving Transfers
* Ticket /bcgov/entity#14103 - Fixed Draft Transfer with group removals

Screen Shot 2022-11-04 at 10 16 25 AM
Screen Shot 2022-11-04 at 10 16 03 AM
Screen Shot 2022-11-04 at 10 15 48 AM

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

File: ppr-ui/src/components/common/HomeLocationInfo.vue [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+        districtLot: '',
+        partOf: '',
+        section: '',
+        township: '',
+        range: '',
+        meridian: '',
+        parcel: '',
+        block: '',
+        exceptPlan: ''
+      },
```
Comment (dimak1) [inline ppr-ui/src/components/common/HomeLocationInfo.vue]: Let's add a type here `MhrLocationInfoIF`.

**Review by dimak1 — APPROVED** [review]
Approved with minor comment.

File: ppr-ui/src/components/common/HomeLocationInfo.vue (hunk 1 above)
Comment (cameron-eyds) [reply ppr-ui/src/components/common/HomeLocationInfo.vue]: For sure!

Comment (cameron-eyds) [reply ppr-ui/src/components/common/HomeLocationInfo.vue]: updated

**Pull request merged**